features/functionality/users/list.feature

```diff
@@ -1,10 +1,9 @@
 Feature: Users List
-  As a user 
+  As a admin user
   I want to see users registered in system
   So I can see Users list
 
   Scenario: user is viewing users in Users list
-    Given there is registered user
```

Comment:
As far as I remember previously we had already have admin user in this step. As result, we faced with race conditions when rspec, turnip and cucumber tests were executed in parallel. So, please roll back your change to new user creation here!

Reply:
But usual user doesn't have access to user list